src/apps/project-settings/ProjectSettings.tsx

```diff
@@ -1,70 +1,116 @@
 import { useEffect, useState } from 'react';
-import { clearProjectTagVocabulary, getProjectTagVocabulary, setProjectTagVocabulary } from '@backend/helpers';
+import {
```

Comment:
We should definitely also refactor this at some point. Project access settings and the tag vocabulary are separate things, and we should pack them into separate components. Perhaps wait for Chelseas review first, but we should IMO put it on the list.